src/main/java/edu/ucsb/courses/documents/statistics/SingleCourseSearch.java

```diff
@@ -0,0 +1,78 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SingleCourseSearch {
+
+    private static Logger logger = LoggerFactory.getLogger(QuarterDept.class);
```

Comment:
Should this be `LoggerFactory.getLogger(SingleCourseSearch.class);`?

Reply:
This has been fixed.

Reply:
We changed it to SingleCourseSearch